test/e2e/data/shared/v1beta2_provider/metadata.yaml

```diff
@@ -41,3 +41,27 @@ releaseSeries:
   - major: 2
     minor: 0
     contract: v1beta1
+  - major: 2
+    minor: 1
+    contract: v1beta1
+  - major: 2
+    minor: 2
+    contract: v1beta1
+  - major: 2
+    minor: 3
+    contract: v1beta1
+  - major: 2
+    minor: 4
+    contract: v1beta1
+  - major: 2
+    minor: 5
+    contract: v1beta1
+  - major: 2
+    minor: 6
+    contract: v1beta1
+  - major: 2
+    minor: 7
+    contract: v1beta1
+  - major: 9
+    minor: 9
+    contract: v1beta1
```

Comment:
Is this a test or is this going to stay?

Reply:
Its going to stay. It denotes the "next" version which really is what is on "main". We previously had it as a 2.0 version but i felt we needed something a bity more explicit.

Reply:
Ah nice improvement. Thanks!